examples/prototypeApplication/data/worksToTrees.ts

```diff
@@ -212,6 +212,7 @@ export const worksToTreesPrototype: PrototypeApplication = {
           },
         },
         area: {
+          hectares: 0.013942,
```

Comment:
Neither of these were actual validation erorrs, just minor tweaks for consistency / quality of example data !